src/content/1.7/development/webservice/tutorials/2-testing-access.md

```diff
@@ -0,0 +1,164 @@
+The products</description>
+<schema xlink:href="http://example.com/api/products?schema=blank" type="blank"/>
+<schema xlink:href="http://example.com/api/products?schema=synopsis" type="synopsis"/>
+</products>
+</api>
+</prestashop>
+```
+
+## Resource schemas
+
+You can also see that each resource provides two link to access its schemas:
+
+- **blank schema** which you can use as a blank content to create a resource
+- **synopsis schema** which is the same blank schema with additional details on each field
+
+### Blank schema
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<prestashop xmlns:xlink="http://www.w3.org/1999/xlink">
+<address>
+	<id></id>
+	<id_customer></id_customer>
+	<id_manufacturer></id_manufacturer>
+	<id_supplier></id_supplier>
+	<id_warehouse></id_warehouse>
+	<id_country></id_country>
+	<id_state></id_state>
+	<alias></alias>
+	<company></company>
+	<lastname></lastname>
+	<firstname></firstname>
+	<vat_number></vat_number>
+	<address1></address1>
+	<address2></address2>
+	<postcode></postcode>
+	<city></city>
+	<other></other>
+	<phone></phone>
+	<phone_mobile></phone_mobile>
+	<dni></dni>
+	<deleted></deleted>
+	<date_add></date_add>
+	<date_upd></date_upd>
+</address>
+</prestashop>
+```
+
+### Synopsis schema
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<prestashop xmlns:xlink="http://www.w3.org/1999/xlink">
+<address>
+	<id_customer format="isNullOrUnsignedId"></id_customer>
+	<id_manufacturer format="isNullOrUnsignedId"></id_manufacturer>
+	<id_supplier format="isNullOrUnsignedId"></id_supplier>
+	<id_warehouse format="isNullOrUnsignedId"></id_warehouse>
+	<id_country required="true" format="isUnsignedId"></id_country>
+	<id_state format="isNullOrUnsignedId"></id_state>
+	<alias required="true" maxSize="32" format="isGenericName"></alias>
+	<company maxSize="255" format="isGenericName"></company>
+	<lastname required="true" maxSize="255" format="isName"></lastname>
+	<firstname required="true" maxSize="255" format="isName"></firstname>
+	<vat_number format="isGenericName"></vat_number>
+	<address1 required="true" maxSize="128" format="isAddress"></address1>
+	<address2 maxSize="128" format="isAddress"></address2>
+	<postcode maxSize="12" format="isPostCode"></postcode>
+	<city required="true" maxSize="64" format="isCityName"></city>
+	<other maxSize="300" format="isMessage"></other>
+	<phone maxSize="32" format="isPhoneNumber"></phone>
+	<phone_mobile maxSize="32" format="isPhoneNumber"></phone_mobile>
+	<dni maxSize="16" format="isDniLite"></dni>
+	<deleted format="isBool"></deleted>
+	<date_add format="isDate"></date_add>
+	<date_upd format="isDate"></date_upd>
+</address>
+</prestashop>
+```
+
+## JSON format
+
+The Web services can also output JSON instead of XML. To enable JSON output you have two choices:
+
+### Query parameter
+
+Add one of the following parameters to your query string:
+
+- `output_format=JSON`
+- `io_format=JSON`
```

Comment:
```suggestion
- `output_format=json`
- `io_format=json`
```

use lowercase instead, we will follow the Symfony type in the future, even if it's `JSON` in the code, there is no strict comparison :)

Reply:
Well in this case seems like there is, the lowercase doesn't work